test/UnitTests/TestFramework.UnitTests/Assertions/AssertTests.IComparableTests.cs

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using TestFramework.ForTestingMSTest;
+
+namespace Microsoft.VisualStudio.TestPlatform.TestFramework.UnitTests;
+
+public partial class AssertTests : TestContainer
```

Comment:
For all tests where we expect an issue, please do check the full message string instead of only parts of it (contains)